lighthouse-core/closure/third_party/google_universal_analytics_api.js

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2015 The Closure Compiler Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Externs for the Google Universal Analytics API (analytics.js).
+ *
+ * The analytics.js API allows the user to choose a name of Google Analytics
+ * object.  This externs file assumes that the name is ‘ga’.  If you include
+ * analytics.js snippet and use a different name, this externs file will not
+ * be useful to you (unless you change the name ‘ga’ to whatever name you’ve
+ * chosen).
+ *
+ * @see https://developers.google.com/analytics/devguides/collection/analyticsjs/method-reference
+ * @externs
+ */
+
+
+/**
+ * @param {string} methodName
+ * @param {...?} var_args
+ * @return {?}
+ */
+function ga(methodName, var_args) {}
```

Comment:
I don't think putting functions on this function will actually work. Is the rest of this file needed right now?

Definitely should avoid unknown type and do `...*` for `var_args` and no return type if it's not needed. Right now the second argument is being used as `!Object<string, *>` and not any more arguments after that. Is that better?